src/pagination.rs

```diff
@@ -262,12 +384,24 @@ fn interpret_permalink(
     config: &PaginationConfig,
     doc: &Document,
     index_num: usize,
+    extra_include: Option<&str>,
 ) -> Result<String> {
     Ok(if index_num == 1 {
-        doc.url_path.clone()
+        if let Some(extra_include) = extra_include {
+            let include: &str = config.include.into();
+            format!("{}/{}", include, slug::slugify(extra_include))
+        } else {
+            doc.url_path.clone()
+        }
     } else {
         let mut attributes = document::permalink_attributes(&doc.front, &doc.file_path);
-        let pagination_attr = pagination_attributes(index_num as i32, config.include);
+        let include: &str = config.include.into();
+        let include: String = if let Some(extra_include) = extra_include {
+            format!("{}/{}", include, slug::slugify(extra_include))
```

Comment:
`include` is the one defined in the RFC, can only be `All` on master. This PR aims to add `Tags`. When using `Tags`, we create a directory call `tags` and `extra_include` represents the tag so we got this directory structure:
```
_site
   |__posts
   |__all <-- only if we have an index with `include: All`
   |__tags
         |__tag1
         |     |__ index.html
         |     |__ _p <-- only if nb of posts with tag1 above `per_page`
         |           |__ 2
         |                 |__ index.html
         |__tag2
```

Reply:
But why are we taking on the policy to add `include` and `extra_include` to the path rather than offering them as variables to the user and they control it?

Reply:
Now also trying to figure out what is a good name for `extra_include`.  We should probably use that same name for `index_title`.

Reply:
> But why are we taking on the policy to add include and extra_include to the path rather than offering them as variables to the user and they control it?

Do you have an example of such variable so I can try to figure out a way to do that. It's true I didn't think of this way to do it. I was coding what we wrote in the RFC :)

Reply:
I'm still trying to find a better name here. `extra_include` is not good at all, I agree.
`include_postfix` is more understandable, but it does not reflect the `index_title` usage.

or maybe use `index_title` here as well?

Reply:
I decided to use `index_title` :)